app/services/admin_creator_service.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class AdminCreatorService < ApplicationService
+  initialize_with :user
+
+  def call
+    user.update(admin: user_first?)
+  end
+
+  private
+
+  def user_first?
+    User.limit(2).count == 1
```

Comment:
ahaha, that `.limit(2)` doesn't optimize anything :) It makes a query even more complex
This whole checking is expensive only because we have to make it every signup, but `count` itself is super cheap